sdk/core/azure-core/src/environment_log_level_listener.cpp

```diff
@@ -120,11 +120,30 @@ EnvironmentLogLevelListener::GetLogListener()
 
   static std::function<void(Logger::Level level, std::string const& message)> const consoleLogger =
       [](auto level, auto message) {
-        std::cerr << '['
-                  << Azure::DateTime(std::chrono::system_clock::now())
-                         .ToString(
-                             DateTime::DateFormat::Rfc3339, DateTime::TimeFractionFormat::AllDigits)
-                  << "] " << LogLevelToConsoleString(level) << " : " << message << std::endl;
+        std::ostream* os = &std::cout;
```

Comment:
This should be `cerr`.
Cout is for program output.
Cerr is for diagnostic messages  - not only errors. We are breaking that, plus if we release this, we should update changelog.
Customers won't like that we are messing with their output.
We have readmes detailing how to enable logging and redirect `cerr` to logs.

https://en.wikipedia.org/wiki/Standard_streams#Standard_error_(stderr)

https://en.cppreference.com/w/cpp/io/c/std_streams

Reply:
That is fair, reverting.